### PR TITLE
[event_v2] migrate move-examples to events v2

### PR DESCRIPTION
**Pull request by lightmark**
### Description

part of #10529 
### Test Plan
ut

### CODE REVIEW

File: aptos-move/move-examples/marketplace/sources/events.move [hunk 1]
```diff
@@ -17,46 +16,10 @@ module marketplace::events {
     friend marketplace::listing;
     friend marketplace::token_offer;
 
-    /// Marketplace does not have EventsV1
+    /// Marketplace does not have sV1
```
Comment (movekevin) [inline aptos-move/move-examples/marketplace/sources/events.move:19]: Huh?

File: aptos-move/move-examples/marketplace/sources/fee_schedule.move [hunk 2]
```diff
@@ -32,7 +30,7 @@ module marketplace::fee_schedule {
         /// Ref for changing the configuration of the marketplace
         extend_ref: ExtendRef,
         /// An event stream of changes to the fee schedule
-        mutation_events: EventHandle<MutationEvent>,
+        mutation_events: EventHandle<Mutation>,
```
Comment (movekevin) [inline aptos-move/move-examples/marketplace/sources/fee_schedule.move:33]: This should be removed right?

File: aptos-move/move-examples/swap/sources/liquidity_pool.move [hunk 3]
```diff
@@ -76,7 +76,7 @@ module swap::liquidity_pool {
         pending_pauser: address,
         stable_fee_bps: u64,
         volatile_fee_bps: u64,
-        create_pool_events: EventHandle<CreatePoolEvent>,
+        create_pool_events: EventHandle<CreatePool>,
     }
```
Comment (movekevin) [inline aptos-move/move-examples/swap/sources/liquidity_pool.move:80]: Shouldn’t this be removed?

**Review by movekevin — APPROVED** [review]
So much cleaner! Can you make sure there are no event handles left behind? I still saw a few.

**Review by xbtmatt — APPROVED** [review]

**Review by jp-hammer — COMMENTED** [review]
@jp-hammer

**Pull request merged**